### PR TITLE
CSHARP-4152: Add createCollection and collMod spec tests for changeStreamPreAndPostImages option

### PR DESCRIPTION
**Pull request by BorisDog**
`modifyCollection-pre_and_post_images` skipped as we don't have the corresponding helpers.

### CODE REVIEW

**Review by JamesKovacs — CHANGES_REQUESTED** [review]
Missing one test.

File: tests/MongoDB.Driver.Tests/MongoDatabaseImplTests.cs [hunk 1]
```diff
@@ -531,6 +532,7 @@ public void Settings_should_be_set()
             op.AutoIndexId.Should().Be(options.AutoIndexId);
 #pragma warning restore
             op.Capped.Should().Be(options.Capped);
+            op.ChangeStreamPreAndPostImages.Should().BeNull();
```
[the remark this answers is not in the document]
Comment (BorisDog) [reply tests/MongoDB.Driver.Tests/MongoDatabaseImplTests.cs:541]: Done.

**Review by JamesKovacs — APPROVED** [review]
LGTM

**Comment by BorisDog** [comment]
[Pushed](https://github.com/mongodb/mongo-csharp-driver/commit/826d8db16bc39278f0373ddab1e91b30e518f7b4)

**Comment by BorisDog** [comment]
[Pushed](https://github.com/mongodb/mongo-csharp-driver/commit/826d8db16bc39278f0373ddab1e91b30e518f7b4)

**Pull request closed without merging**